thing-url-adapter.js

```diff
@@ -264,8 +283,20 @@ class ThingURLDevice extends Device {
     if (this.closing) {
       return;
     }
+    // if we have an entry for the wss url stub in jwt_auth
+    // add auth to query string
+    let auth = '';
+    for (const i in JWT_AUTH) {
+      if (this.wsUrl.includes(i)) {
+        auth = `?Authorization=Bearer ${JWT_AUTH[i]}`;
```

Comment:
This is a bit of a strange header. What we do for the gateway WebSockets is `jwt=<token>`, without the `Bearer`.

Reply:
I was thinking that doing it this way would save me some effort -- only have to make one string, etc. Didn't really seem to save a lot. The backend code that checks the auth on the rest calls check this on an upgrade so I run it through the header tests that are available. 

I think I'll go with your way -- ?jwt=token